api/lambdas/sparql.ts

```diff
@@ -0,0 +1,99 @@
+import got from 'got';
+import type {
+  APIGatewayProxyEventV2,
+  APIGatewayProxyResult,
+} from 'aws-lambda';
+import { URL } from 'url';
+import { pick } from 'lodash';
+
+export const getStringFromEnv = (key: string): string | Error =>
+  process.env[key] || new Error(`${key} not set`);
+
+export const okResponse = (
+  contentType: string,
+  body: string
+): APIGatewayProxyResult => ({
+  statusCode: 200,
+  headers: { 'Content-Type': contentType },
```

Comment:
Since this is an API Gateway proxy request I think we need to make sure the CORS headers are explicitly set.

Reply:
Setting `Access-Control-Allow-Origin` header: a6d431a